src/extra/imgui.cpp

```diff
@@ -537,10 +537,7 @@ struct Window : public Base {
   std::string _title;
   bool firstActivation{true};
   Var _pos{}, _width{}, _height{};
-  bool _movable{false};
-  bool _closable{false};
-  bool _resizable{false};
-  bool _showMenuBar{false};
+  ParamVar _flags;
```

Comment:
Are the defaults good?

Reply:
Good question. The defaults are now "moveable", "resizeable" and "collapsible".

That's because I followed the same flags (and same values) as in ImGui since it's simpler.

If we want different defaults (and for instance invert the logic around move, collapse and resize) then we need to create independent flags and have a conversion function. That implies a lot of if/else potentially which I wanted to avoid.

Reply:
On second thought, let's go with just better defaults, as long as there are properly documented. On that note, we need to come up with something better when the values of enum are generated in the documentation.